packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -91,6 +91,8 @@ export class MLSService extends TypedEventEmitter<Events> {
       keyingMaterialUpdateThreshold = defaultConfig.keyingMaterialUpdateThreshold,
       nbKeyPackages = defaultConfig.nbKeyPackages,
     }: Partial<MLSServiceConfig>,
+    private readonly textEncoder = new TextEncoder(),
+    private readonly textDecoder = new TextDecoder(),
```

Comment:
No need to put them as params of the constructor, you could move them a few lines above

```js
class MLSService {
    logger = logdown(..)
    //...
    private readonly textEncoder = new TextEncoder();
    private readonly textDecoder = new TextDecoder();

   constructor() //..
}
```